cli.js

```diff
@@ -116,6 +116,11 @@ if (cli.flags.init) {
 	return;
 }
 
+if (cli.flags.watch && cli.flags.tap && !conf.tap) {
```

Comment:
This is better. However, if they do `--tap --watch`, AND have `tap:true` in their config they won't get this error message.

The only thing I can think of is to use `hasFlag('--tap')` instead of `cli.flags.tap`.

Also, they could specify `{watch: true, tap: true}` in their config and not get this warning.

So maybe:

```js
if ((hasFlag('--tap') && hasFlag('--watch')) || (conf.watch && conf.tap)) {
  // ...
}
```

... Or I'm just overthinking it.